src/screens/ForgetPasswordScreen/ForgetPasswordScreen.tsx

```diff
@@ -3,12 +3,10 @@ import React, {useEffect, useState} from "react";
 import {useDispatch, useSelector} from "react-redux";
 import {forgetPassword} from "../../redux/actions/forgetPassword";
 import {useNavigation} from "@react-navigation/native";
-import {CodeVerify} from "../CodeVerifyScreen/CodeVerify";
 import {emailValidationSchema} from "../LoginScreen/validationScheme";
 import {Formik} from "formik";
 import {styles} from "./styles";
 import {Loader} from "../../components/Loader";
-import Animated, {} from "react-native-reanimated";
 import {displayErrorMessage} from "../../utils/displayMessage";
 import {RootState} from "../../redux/reducers/rootReducer";
 import {AnimatedText} from "./AnimatedText";
```

Comment:
Also this component uses `useEffect` to display error message on error case. The error comes from redux. If the user leaves the screen and then returns back to it, the error message will be displayed again as it's still in store. That's a bug.

Moreover this "display error message if error occurs" logic is going to be repetitive, we should think of a more common solution. Let's make it our next task, no need for this in that PR